### PR TITLE
some minor improvements to the built-in fake

### PR DESCRIPTION
**Pull request by novas0x2a**

### CODE REVIEW

**Review by bacongobbler — APPROVED** [review]

File: pkg/helm/fake.go [hunk 1]
```diff
@@ -97,7 +97,9 @@ func (c *FakeClient) InstallReleaseFromChart(chart *chart.Chart, ns string, opts
 	}
 
 	release := ReleaseMock(&MockReleaseOptions{Name: releaseName, Namespace: ns, Description: releaseDescription})
-	c.Rels = append(c.Rels, release)
+	if !c.Opts.dryRun {
+		c.Rels = append(c.Rels, release)
```
Comment (bacongobbler) [inline pkg/helm/fake.go]: I think we still want to display the "deployed" release in the response, no?

**Review by bacongobbler — CHANGES_REQUESTED** [review]

File: pkg/helm/fake.go (hunk 1 above)
Comment (novas0x2a) [reply pkg/helm/fake.go]: Hm, not sure I understand what you mean; with this change, a dry-run release is returned in the InstallReleaseResponse but isn't remembered by the "server", same as real code.

Comment (bacongobbler) [reply pkg/helm/fake.go]: oh gotcha. That's what I get for reading code prior to coffee. ☕️

Comment (bacongobbler) [reply pkg/helm/fake.go]: oh gotcha. That's what I get for reading code prior to coffee consumption. ☕️

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**